lib/mongo_conn.js

```diff
@@ -0,0 +1,60 @@
+/* eslint no-process-env: "off" */
+const { MongoClient } = require('mongodb');
+let hosts;
+let mongo;
+if (process.env.AMK_MONGO_HOSTS) {
+    const _hosts = [];
+    hosts = process.env.AMK_MONGO_HOSTS.split(',');
+    _hosts.push(hosts);
+    hosts = _hosts;
+}
+
+const conn = {
+    hosts: hosts || ['127.0.0.1:27017']
+};
+
+function connect(config) {
+    if (mongo) {
+        return mongo;
+    }
+    let url = 'mongodb://';
+    let hosts = (config && config.hosts) || conn.hosts;
+    let host = hosts.join(',');
+    if (config && config.user) {
+        url += config.user + ':';
+    } else if (process.env.AMK_MONGO_USERNAME) {
+        url += process.env.AMK_MONGO_USERNAME + ':';
+    }
+
+    if (config && config.password) {
+        url += config.password + '@';
+    } else if (process.env.AMK_MONGO_PASSWORD) {
+        url += process.env.AMK_MONGO_PASSWORD + '@';
+    }
+
+    url += host + '/';
+
+    if (config && config.database) {
+        url += config.database;
+    } else if (process.env.AMK_MONGO_DEFAULT_DATABASE) {
+        url += process.env.AMK_MONGO_DEFAULT_DATABASE;
+    }
+
+    let option = {
+        useNewUrlParser: true
+    };
+
+    if (config && config.option) {
+        Object.assign(option, config.option);
+    }
+
+    return MongoClient.connect(url, option);
+}
+
+module.exports = connect;
+exports.config = {
+    AMK_MONGO_DEFAULT_DATABASE: process.env.AMK_MONGO_DEFAULT_DATABASE,
+    AMK_MONGO_HOSTS: process.env.AMK_MONGO_HOSTS,
+    AMK_MONGO_USERNAME: process.env.AMK_MONGO_USERNAME,
+    AMK_MONGO_PASSWORD: process.env.AMK_MONGO_PASSWORD
+};
```

Comment:
should return a new line.

Reply:
why need a new line ?

Reply:
I think it is part of eslint convention to have a new line at the end of file.